shared/src/business/entities/externalDocument/ExternalDocumentNonStandardD.js

```diff
@@ -37,14 +40,14 @@ ExternalDocumentNonStandardD.VALIDATION_ERROR_MESSAGES = {
 };
 
 ExternalDocumentNonStandardD.schema = {
-  category: joi.string().required(),
```

Comment:
Similar to CourtIssuedDocumentType{letter}, I wonder if we could consolidate the common attributes in these different ExternalDocumentNonStandard{letter} entity files, so you can see the difference between them at a glance.

Reply:
From previous comment:
> `attachments`, `documentType`, and `documentTitle` are about the only fields that are shared across these Entities, and some of them are handled differently in the constructor (for example, `documentTitle` is usually composed of other fields on the Entity). I'll add a DevEx card to Trello to discuss this.